apps/system/js/nfc_handover_manager.js

```diff
@@ -132,6 +132,15 @@
       this.bluetoothStatusSaved = false;
       this.bluetoothAutoEnabled = false;
 
+      // Besides listening to bluetooth-enabled event,
+      // request the adapter at module start to 
+      // avoid the module loading dependency
+      Service.request('Bluetooth:adapter').then((adapter) => {
```

Comment:
According to bluetooth.js, this promise never resolves if Bluetooth is not turned on, and it will only resolve when it is later turned on. So there is no conflict between this and https://github.com/gasolin/gaia/blob/issue-1183496/apps/system/js/nfc_handover_manager.js#L169